core/pta/tests/misc.c

```diff
@@ -391,7 +432,7 @@ static int self_test_nex_malloc(void)
 	bool r;
 	int ret = 0;
 
-	LOG("nex_malloc tests (nex_malloc, nex_free, nex_calloc, nex_realloc):");
+	LOG("nex_malloc tests (nex_malloc, nex_free, nex_calloc, nex_realloc, nex_memalign):");
```

Comment:
Since checkpatch complains on this, the function list had better be removed IMO.

Reply:
OK

Reply:
Thanks! As you can see I'm taking care of the bikeshedding on this one :laughing:

Reply:
Haha, an ungrateful task. :-)